atlp-rwanda/e-comm-team-brogrammers-fn
# Users(Sellers) should be able to turn on/off MFA

#### What does this PR do?
This PR provides the ability for users to increase level of security by enabling MFA for their accounts.
#### Description of Task to be completed?

Users that require a higher level of security (Sellers) should be able to turn on 2FA where authentication to the application also includes a an extra factor i.e SMS or Authentications codes. 
After a successful login, Sellers should be redirected to their dashboard

#### How should this be manually tested?
**To test this: **
- Login as usual
- Go to `/settings`
- Click Enable MFA button to enable MFA
- Logout
- When logging in again you will be asked to provide MFA code
- Enter the code from email, and hit verify. 
- You should be logged in.
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/n/projects/2639400/stories/184948160
#### Screenshots (if appropriate)
![Uploading Screenshot 2023-05-11 at 14.16.57.png…]()

#### Questions:
n/a
